src/elements/content-sidebar/DocGenSidebar/TagTree.tsx

```diff
@@ -13,16 +16,25 @@ const TagTree = ({ data, level = 0 }: TagTreeProps) => {
     }
 
     return (
-        <div>
+        <Accordion
+            type='multiple'
+            className="bcs-DocGen-accordion"
+        >
             {Object.keys(data)
                 .sort()
-                .map(key => (
-                    <div key={`${key}-${level}`} style={{ paddingLeft: `${level * 12}px` }}>
-                        <span className="bcs-DocGen-tagPath">{key}</span>
-                        {data[key] && <TagTree data={data[key]} level={level + 1} />}
-                    </div>
-                ))}
-        </div>
+                .map(key => {
+                    if (isEmpty(data[key])) {
+                        return(
+                            <Accordion.Item value={key} key={`${key}-${level}`} style={{ paddingLeft: `${level * 12}px` }} fixed className="bcs-DocGen-collapsible">
+                                <span className="bcs-DocGen-tagPath">{key}</span>
+                            </Accordion.Item>
+                        )
+                    }
+                    return (<Accordion.Item value={key} title={key} key={`${key}-${level}`} style={{ paddingLeft: `${level * 12}px` }} className="bcs-DocGen-collapsible">
```

Comment:
is there a limit to how many levels there could be? is there a point where the item will get pushed off the component? does the text start to wrap it it goes to far to the right ?

Reply:
there's no limit to the number of levels - it's a nested json structure, but good point, will raise with product

Reply:
I've confirmed with product that the behavior is as expected - arbitrarily deep nesting is allowed with horizontal scroll to navigate it.